core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -224,7 +240,8 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
         //Load a bunch of things we'll need to reconstruct information:
         //The list of workflows in this submission
         wfRecs <- getWorkflowRecordBatch(workflowIds, dataAccess)
-        workflowBatch <- reifyWorkflowRecords(wfRecs, dataAccess)
+        workflowBatchRaw <- reifyWorkflowRecords(wfRecs, dataAccess)
+        workflowBatch <- resolveDosUris(workflowBatchRaw)
```

Comment:
This could go down in the `yield` instead somehow (if there's reason for that) as long as it happens before `MethodConfigResolver.propertiesToWdlInputs()`.